src/App.tsx

```diff
@@ -66,4 +72,18 @@ const App: React.FC = () => (
   </>
 )
 
-export default hot(App)
+export default hot(
+  withGlobalContext(
+    App,
+    // Initial State
+    () => {
+      // Make sure key names here match that of src/reducers-actions/combineReducers keys
+      // These key names are the state slices accessible from the useGlobalState hook via
+      // const [{ TokenRow }, dispatch] = useGlobalState()
+      return {
+        TokenRow,
```

Comment:
I don't understand, why the global state is modeled as a TokenRow

Reply:
Okay, I see `TokenRow` is not what I think, we have a component called `TokenRow` but this seems is totally different thing. it's a `TokenRowInitialState` so I wouldn't change the name of that when you import it.

Also, I wouldn't capitalize the state, right? 
```
return {
    tokenRow: TokenRowInitialState
}
```

Actually, the name it's kind of misleading too. Maybe you call that state prop `TokenRow` because you know the data is used in the token row components? I think it's a better name `tokens` for containing the global state related to the tokens (for now, only flags for enabling, highlighting and so on)

Reply:
fixed